test/tests/haskell-runhaskell/container.hs

```diff
@@ -0,0 +1 @@
+main = putStrLn "Hello, World"
```

Comment:
@Moghedrin shouldn't this test have an `expected-std-out.txt` to verify that it runs and outputs `Hello, World` appropriately?